bodhi/server/schemas.py

```diff
@@ -653,10 +659,11 @@ class ListUpdateSchema(PaginatedSchema, SearchableSchema, Cosmetics):
         validator=colander.OneOf(list(ContentType.values())),
     )
 
-    user = colander.SchemaNode(
-        colander.String(),
+    user = Users(
+        colander.Sequence(accept_scalar=True),
         location="querystring",
         missing=None,
+        preparer=[util.splitter],
```

Comment:
It is not necessary to support splitting the users by commas, as you can just specify the ```user``` parameter multiple times in a single GET request. For example, here's a query for comments that exclude a bunch of users:

http://localhost:6543/comments/?ignore_user=labbott&ignore_user=elyscape&ignore_user=kevin&ignore_user=bowlofeggs&ignore_user=bodhi

Reply:
Fair enough.